portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase

```diff
@@ -19,58 +19,82 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LRQA-41349. Verify that a site admin could delete a Custom Export."
```

Comment:
e.g.
@description = "Admin user can delete the custom export process."

In this description, we focused on two personas that are admin user and export process. Not only mention one user or one process.